pym/portage/dbapi/bintree.py

```diff
@@ -568,7 +579,7 @@ def _populate(self, getbinpkgs=False, getbinpkg_refresh=True):
 				metadata[_instance_key(cpv)] = d
 				path = d.get("PATH")
 				if not path:
-					path = cpv + ".tbz2"
+					path = cpv + "." + self.compression["suffix"]
```

Comment:
My plan was to use the `.xpak` suffix for all types of compression. The `.xpak` extension is already used when `FEATURES=binpkg-multi-instance` is enabled. We can use it here too.